.github/workflows/CI.yml

```diff
@@ -215,23 +215,24 @@ jobs:
         run: mkdir stable_regression_results
 
       # Retrieve Stable Results for reference
-      # Will need to use this -> https://github.com/dawidd6/action-download-artifact
+      - name : ChatGPT said this would work - Get most recent CI run ID
```

Comment:
I suggest remove the chatGPT bit

Reply:
Haha forgot i had that in there, removing now